### PR TITLE
apps/examples/elf : Add break in wifiapp.c and change priority in /micomapp/Messaging.c, os/kernel/messaging : modify messaging_append_receiver

### PR DESCRIPTION
**Pull request by hasw7569**
- apps/examples/elf : Add break in wifiapp.c and change priority in /micomapp/Messaging.c
- os/kernel/messaging : modify messaging_append_receiver

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [71eadc7398961ba7170bdadd492526f569870f5b] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [71eadc7398961ba7170bdadd492526f569870f5b] - Code Rule Check OK.

File: apps/examples/elf/wifiapp/wifiapp.c [hunk 1]
```diff
@@ -68,6 +68,7 @@ int main(int argc, char **argv)
 		case 'x':
 			printf("Test will be finished.\n");
 			is_testing = false;
+			break;
```
Comment (sunghan-chang) [inline apps/examples/elf/wifiapp/wifiapp.c]: Please revert this line. This was merged with commit https://github.com/Samsung/TizenRT/commit/c1fcc954f0627b5165282c8fc96a312ca856c7a2

**Comment by seinfra** [comment]
Target : [c7f60d96e5e727a219b83445940eee20c8327464] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [c7f60d96e5e727a219b83445940eee20c8327464] - Code Rule Check OK.

File: apps/examples/elf/wifiapp/wifiapp.c (hunk 1 above)
Comment (hasw7569) [reply apps/examples/elf/wifiapp/wifiapp.c]: modify

**Pull request merged**